bitbots_hardware_rqt/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ catkin_python_setup()
 
 add_message_files(
    FILES
-   BatteryMessage.msg
```

Comment:
Catkin refuses to build when no message files are listed.
Please remove `add_message_files`, `generate_messages` and dependencies which are then not needed anymore

Reply:
Should be fixed now.